### PR TITLE
🌱 Bump CAPI to v1.8.1

### PR DESCRIPTION
**Pull request by Sunnatillo**
Uplifts CAPI to v1.8.1

Part of: https://github.com/metal3-io/cluster-api-provider-metal3/issues/1887

### CODE REVIEW

**Review by tuminoid — COMMENTED** [review]
/approve

**Comment by adilGhaffarDev** [comment]
can you add in pr description part of https://github.com/metal3-io/cluster-api-provider-metal3/issues/1887

**Review by smoshiur1237 — COMMENTED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -49,9 +49,9 @@ need to be manually installed. Example flow of installing Metal3 provider:
    level of the logging verbose with a positive integer number, ie. -v5.
 
    ```shell
-   clusterctl init --core cluster-api:v1.8.0 \
-       --bootstrap kubeadm:v1.8.0 \
-       --control-plane kubeadm:v1.8.0 -v5
+   clusterctl init --core cluster-api:v1.8.1 \
+       --bootstrap kubeadm:v1.8.1 \
+       --control-plane kubeadm:v1.8.1 -v5
```
Comment (kashifest) [inline README.md:54]: I think we decided not to touch these README for patch releases?

**Comment by kashifest** [comment]
/lgtm
/hold 
Small nit inline

File: README.md (hunk 1 above)
Comment (Sunnatillo) [reply README.md:54]: I do not remember. But why? 
IMHO if we do not change every time we most likely forget to update it.

Comment (tuminoid) [reply README.md:54]: It prevents dependabot from doing these bumps as it cannot handle README.md

README doesn't need to update all the time, so bump once when bumpign minor and then leave alone and let bot do it.

Comment (Sunnatillo) [reply README.md:54]: noted. Next time I will not update it.

**Pull request merged**